gaphor/codegen/tests/test_autocoder.py

```diff
@@ -17,32 +17,33 @@ def close(self):
 
 
 def test_loading():
-    model_file = distribution().locate_file("tests/test-model.gaphor")
+    model_file = distribution().locate_file("gaphor/codegen/tests/test-model.gaphor")
```

Comment:
Should we move the test model to test-diagrams?

I know it's not ideal, since unit tests are packaged with the production code. Now there are several tests already that break because they depend on diagrams outside the source (code) tree. 

Either we should move all these tests out of the source tree into `/tests/` or include all test diagrams in the source tree. I'd opt for the first.

Reply:
Thanks, I agree that we should keep the models out of the source packages. Thanks for catching that! :+1: Since you created a new models package, I thought it made sense to move test-diagrams to test-models and nest it under where the rest of the models are located. What do you think about that?